static/js/vue-cdr-access/src/components/searchWrapper.vue

```diff
@@ -89,14 +89,8 @@
         methods: {
             retrieveData() {
                 let param_string = `${this.formatParamsString(this.$route.query)}&getFacets=true`;
-                let search_path = 'searchJson';
-
-                this.collection = UUID_REGEX.test(this.$route.path) ? this.$route.path.split('/')[2] : '';
```

Comment:
I wonder if we should keep this logic? It allows scoping a search result to containers other than collections, which I personally find kind of helpful, and we might want to expose again later or in conjunction with adding more faceting/search options to the result pages.